cmd/kubeadm/app/cmd/init.go

```diff
@@ -191,6 +192,18 @@ func AddInitConfigFlags(flagSet *flag.FlagSet, cfg *kubeadmapiext.MasterConfigur
 		&cfg.TokenTTL.Duration, "token-ttl", cfg.TokenTTL.Duration,
 		"The duration before the bootstrap token is automatically deleted. If set to '0', the token will never expire.",
 	)
+	flagSet.StringVar(
+		&cfg.AuditPolicyConfiguration.Path, "audit-policy", cfg.AuditPolicyConfiguration.Path,
```

Comment:
I think there was agreement in yesterday's SIG meeting to not proceed with adding flags until we sort out whether we want to be supporting these in the future.

@kubernetes/sig-cluster-lifecycle-misc

Reply:
I think we may have to default off with a opt-in via config option for the time being.  Due to the logistical constraints on both args, and self-hosting HA concerns.  

This will enable folks to turn on b/default with a separate PR, and unblock this one.